model/QueueBroker/RdsQueueBroker.php

```diff
@@ -140,8 +147,10 @@ public function createQueue()
     public function push(TaskInterface $task)
     {
         return (bool) $this->getPersistence()->insert($this->getTableName(), [
+            'id' => $this->getUniquePrimaryKey(),
             'message' => $this->serializeTask($task),
-            'created_at' => $this->getPersistence()->getPlatForm()->getNowExpression()
+            'created_at' => $this->getPersistence()->getPlatForm()->getNowExpression(),
```

Comment:
spanner allows you to use a placeholder may in the improvements we  can add that also

Reply:
Not really get what do you mean, could you expand  your idea

Reply:
`spanner.commit_timestamp()`
is something that spanner knows to handle 
to explain it --> it is something like `CURRENT_TIMESTAMP`

Reply:
anyways just as an improvement, not necessary now